packages/repository/src/common-types.ts

```diff
@@ -99,10 +99,14 @@ export interface Count {
  * REST calls to APIs which return Count
  */
 export const CountSchema = {
-  type: 'object',
+  type: 'object' as const, // Force to be `object` type instead of `string`
```

Comment:
Can we cast `CountSchema` instead of adding `as const`, for consistency with other places?

```ts
export const CountSchema: SchemaObject = {
  type: 'object',
  // ...
};
```

Reply:
Haha, the `@loopback/repository` does not have dependency on `@loopback/openapi-v3`. That's why I forced the duck-typing with the `as const`.

Reply:
I added more ts comments.